File: codes/fizzbuzz.cs
using System;

namespace Program {
  class FizzBuzz {
    static void Main(string[] args) {
      for(int i = 0; i < 100; i++) {
        Console.WriteLine(i % 15 ? "FizzBuzz" : i % 3 ? "Fizz" : i % 5 ? "Buzz" : i)
      }
    }
  }
}

File: codes/fizzbuzz.lua
for i = 1, 100, 1 do
  if i % 15 == 0 then
    print('FizzBuzz')
  end
  if i % 3 == 0 then
    print('Fizz')
  end
  if i % 5 == 0 then
    print('Buzz')
  else
    print(i)
  end
end
